admin/data/sql/migrations/20190430141939.do.drop-updatedat-column-triggers-pupil-table.sql

```diff
@@ -0,0 +1,29 @@
+DECLARE @tableName VARCHAR(max)
+DECLARE @sql NVARCHAR(max)
+
+DECLARE cur CURSOR FOR
+    SELECT t.name
+    FROM SYS.COLUMNS c
+    JOIN SYS.TABLES t
+        ON c.OBJECT_ID = t.OBJECT_ID
+    WHERE c.name LIKE '%updatedAt%'
```

Comment:
i would prefer this to be explicit with an equality operator.  otherwise you could accidentally pick up another column

Reply:
@GuyHarwood 👍